Plugins/NativeEngine/Source/NativeEngine.h

```diff
@@ -384,19 +386,22 @@ namespace Babylon
     {
         static constexpr auto JS_CLASS_NAME = "_NativeEngine";
         static constexpr auto JS_ENGINE_CONSTRUCTOR_NAME = "Engine";
+        static constexpr auto JS_AUTO_RENDER_PROPERTY_NAME = "_AUTO_RENDER";
 
     public:
         NativeEngine(const Napi::CallbackInfo& info);
-        NativeEngine(const Napi::CallbackInfo& info, Plugins::Internal::NativeWindow& nativeWindow);
+        NativeEngine(const Napi::CallbackInfo& info, JsRuntime& runtime, Plugins::Internal::NativeWindow& nativeWindow);
         ~NativeEngine();
 
-        static void InitializeWindow(void* nativeWindowPtr, uint32_t width, uint32_t height);
-        static void DeinitializeWindow();
-        static void Initialize(Napi::Env);
+        static void Initialize(Napi::Env, bool autoRender);
 
         FrameBufferManager& GetFrameBufferManager();
         void Dispatch(std::function<void()>);
-        void EndFrame();
+
+        void ScheduleRender();
+
+        const bool AutomaticRenderingEnabled{};
```

Comment:
Is this always false?

Reply:
No, it's usually true. A per-`Env` constant is set in the JavaScript type at initialization type; the [default value is true](https://github.com/BabylonJS/BabylonNative/pull/345/files?file-filters%5B%5D=.cpp&file-filters%5B%5D=.h&hide-deleted-files=true#diff-f70275df353ecc0e0b28ddcfc891504cR9). Individual instances of `NativeEngine` will then read this value in their constructors, using that to set the `AutomaticRenderingEnabled` value.